script/find_record.py

```diff
@@ -0,0 +1,34 @@
+import json 
+from sys import argv
+
+
+script, JSON_FILE, user_name  = argv
+
+
+#JSON_FILE = "init\persons.json"
+unflat_persons = json.load(open(JSON_FILE, encoding='utf-8'))
+ 
+
+def find_user_name(first_name):
+  list_of_persons = unflat_persons['results']
+  record_found = False
```

Comment:
You don't need this variable.
Do you know why?

Reply:
I do not need this `print('A user with this name does not exist')` so I do not need this variable? I should use return, if is empty it's mean that record don't exist?

Reply:
Yes, you can check if you found anything by checking if the list is empty